prac_08/taxi_simulator.py

```diff
@@ -3,18 +3,36 @@
 from prac_08.taxi import Taxi
 
 MENU = "1) choose taxi\n2) Drive\n3) Bill\n4) Quit"
```

Comment:
I would use what was given to us in the same output
```suggestion
MENU = "q)uit, c)hoose taxi, d)rive"
```